### PR TITLE
Adding `rasa` and `rasa-sdk` from PyPI

### PR DESCRIPTION
**Pull request by sugatoray**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged). `NA`
- [ ] If static libraries are linked in, the license of the static library is packaged. `NA`
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team. `NA`

### CODE REVIEW

**Comment by sugatoray** [comment]
Note on current error (on Linux):

[error source](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=416073&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=989) :no_entry: 

```diff
  Packaging rasa-3.0.0rc3-pyhd8ed1ab_0
  INFO:conda_build.build:Packaging rasa-3.0.0rc3-pyhd8ed1ab_0
  INFO conda_build.build:bundle_conda(1514): Packaging rasa-3.0.0rc3-pyhd8ed1ab_0
  Using pip 21.3.1 from /home/conda/staged-recipes/build_artifacts/rasa-meta_1637471339521/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/python3.8/site-packages/pip (python 3.8)
  Non-user install because user site-packages disabled
  Ignoring indexes: https://pypi.org/simple
  Created temporary directory: /tmp/pip-ephem-wheel-cache-3c6v_uel
  Created temporary directory: /tmp/pip-req-tracker-jpafw81r
  Initialized build tracking at /tmp/pip-req-tracker-jpafw81r
  Created build tracker: /tmp/pip-req-tracker-jpafw81r
  Entered build tracker: /tmp/pip-req-tracker-jpafw81r
  Created temporary directory: /tmp/pip-install-18tmo2ct
  Processing /home/conda/staged-recipes/build_artifacts/rasa-meta_1637471339521/work
    Added file:///home/conda/staged-recipes/build_artifacts/rasa-meta_1637471339521/work to build tracker '/tmp/pip-req-tracker-jpafw81r'
    Created temporary directory: /tmp/pip-build-env-pdjpnfkq
    Created temporary directory: /tmp/pip-standalone-pip-oiqowywa
    Running command /home/conda/staged-recipes/build_artifacts/rasa-meta_1637471339521/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/bin/python /tmp/pip-standalone-pip-oiqowywa/__env_pip__.zip/pip install --ignore-installed --no-user --prefix /tmp/pip-build-env-pdjpnfkq/overlay --no-warn-script-location -v --no-binary :none: --only-binary :none: --no-index -- 'poetry-core>=1.0.4'
    Installing build dependencies: started
    Using pip 21.3.1 from /tmp/pip-standalone-pip-oiqowywa/__env_pip__.zip/pip (python 3.8)
-   ERROR: Could not find a version that satisfies the requirement poetry-core>=1.0.4 (from versions: none)
-   ERROR: No matching distribution found for poetry-core>=1.0.4
    Installing build dependencies: finished with status 'error'

```

**Comment by sugatoray** [comment]
@BastianZim After I started working on packaging `rasa`, I realized it depends on `rasa-sdk`, which also needs packaging for conda-forge. I keep getting this error for `rasa-sdk` that requires `poetry-core` (which is added to the recipe). I have no clue why this might be happening. Would it be possible for you to take a look at it?

Following your suggestion on putting multiple related packages in the same repository, I adoped that approach here.

Please see the error [here (comment)](https://github.com/conda-forge/staged-recipes/pull/17028#issuecomment-974759799).

Recipe for `rasa-sdk`: https://github.dev/sugatoray/staged-recipes/tree/rasa/recipes/rasa-sdk/meta.yaml

**Comment by BastianZim** [comment]
No idea yet unfortunately, but I'll have a look.

File: recipes/rasa-sdk/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 162e72dc3c141d9f5220f3de921be6c902e03f209980a42cdb2cb1ff21f534d6
+  
+
+build:
+  skip: True  # [osx or win]
+  number: {{ build_num }}
+
+outputs:
```
Comment (BastianZim) [inline recipes/rasa-sdk/meta.yaml:18]: Is it only one package? Then no outputs are needed.

File: recipes/rasa-sdk/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    build:
+      skip: True  # [osx or win]
+      noarch: python
+    script: build_base.sh
+    requirements:
+      {% set pyver = ">=3.7,<3.10" %}
+      host:
+        - pip
+        - python {{ pyver }}
+        - poetry-core >=1.0.0
```
Comment (BastianZim) [inline recipes/rasa-sdk/meta.yaml:29]: Not sure why this is not working. I would ask in the general chat again.

File: recipes/rasa-sdk/meta.yaml (hunk 1 above)
Comment (sugatoray) [reply recipes/rasa-sdk/meta.yaml:18]: It's only one package, but (I think) it also offers extras. So, paving the way, for easily adding the extras should the need arise.

File: recipes/rasa-sdk/meta.yaml (hunk 2 above)
Comment (sugatoray) [reply recipes/rasa-sdk/meta.yaml:29]: Thanks. Will do.

**Comment by sugatoray** [comment]
keep this open

**Pull request closed without merging**